### ISSUE TITLE
NPE in JvmErgonomics fails repository plugin tests

### ISSUE THREAD

**Issue by davidkyle** [elastic/elasticsearch#42009]
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+third-party-tests-azure/40/console
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+third-party-tests-gcs/163/console
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+third-party-tests-s3/163/console

``` 
Task :plugins:repository-s3:integTestRunner FAILED
...
12:33:45   Exception in thread "main" java.lang.NullPointerException
12:33:45   	at org.elasticsearch.tools.launchers.JvmErgonomics.extractHeapSize(JvmErgonomics.java:122)
12:33:45   	at org.elasticsearch.tools.launchers.JvmErgonomics.choose(JvmErgonomics.java:58)
12:33:45   	at org.elasticsearch.tools.launchers.JvmOptionsParser.main(JvmOptionsParser.java:89)
12:33:45 
12:33:45 FAILURE: Build failed with an exception.
```

**Comment by davidkyle** [comment]
Appears to be due to #41962, pinging @jasontedor

**Comment by davidkyle** [comment]
I'm not sure on how to label this one, I've gone for the generic core/infra/core

**Comment by davidkyle** [comment]
Different test, same cause https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+artifactory/665/console

**Comment by jasontedor** [comment]
Closed by 7ee541546a7a7db88028b526355120f0c5153a80
